samples/interactive-tutorials/user_environment_setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright 2022 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# set the Google Cloud Project ID
+project_id=$1
+echo "Project ID: $project_id"
+gcloud config set project "$project_id"
+
+timestamp=$(date +%s)
+
+service_account_id="service-acc-$timestamp"
+echo "Service Account: $service_account_id"
+
+# create service account (your service-acc-$timestamp)
+gcloud iam service-accounts create "$service_account_id"
+
+# assign necessary roles to your new service account
+for role in {retail.admin,editor,bigquery.admin}
+  do
+    gcloud projects add-iam-policy-binding "$project_id" --member="serviceAccount:$service_account_id@$project_id.iam.gserviceaccount.com" --role=roles/"${role}"
+done
+
+echo "Wait ~60 seconds to be sure the appropriate roles have been assigned to your service account"
+sleep 60
+
+# upload your service account key file
+service_acc_email="$service_account_id@$project_id.iam.gserviceaccount.com"
+gcloud iam service-accounts keys create ~/key.json --iam-account "$service_acc_email"
+
+# activate the service account using the key
```

Comment:
Ok, if your doing this, then why tell the user to provide a service account in the README?

Reply:
In the tutorials we give the user 2 options - eanther users want to setup the work env by themselfs, then we provide steps similar to those discribed in the README, or they want to speedup the process and start exploring the Retail features - for tha purpose we give them this scripts. 

But you are right, it is good to tell users that they may use the scripts in the Readme as well